File: mouse_jiggle.js
async function mouseJiggle() {
  try {
    while (true) {
      const xOffset = Math.floor(Math.random() * 10) - 5;
      const yOffset = Math.floor(Math.random() * 10) - 5;

      console.log(`Simulating mouse jiggle: offset x=${xOffset}, y=${yOffset}`);

      // Wait for a random interval between 1 and 5 seconds
      const delay = Math.floor(Math.random() * 4000) + 1000; // milliseconds
      await new Promise(resolve => setTimeout(resolve, delay));
    }
  } catch (err) {
    console.log("Mouse jiggle stopped.");
  }
}

mouseJiggle();

File: mouse_jiggle.py
import random
import asyncio

async def mouse_jiggle():
    try:
        while True:
            # Simulate a small mouse movement
            x_offset = random.randint(-5, 5)
            y_offset = random.randint(-5, 5)

            print(f"Simulating mouse jiggle: offset x={x_offset}, y={y_offset}")
            
            # Wait for a random interval between 1 and 5 seconds
            await asyncio.sleep(random.randint(1, 5))
    except KeyboardInterrupt:
        print("Mouse jiggle stopped.")

if __name__ == "__main__":
    asyncio.run(mouse_jiggle())
